### PR TITLE
Documentation Updates

### PR DESCRIPTION
**Pull request by dellis1972**
Rework/Cleanup the Documentation to reflect the current state
of the project. i.e Remove Playstation Mobile and Windows 8/8.1.

### CODE REVIEW

File: Documentation/config.xml [hunk 1]
```diff
@@ -28,7 +28,9 @@
         <topic id="MonoGame_FAQ" name="MonoGame FAQ" filename="monogame_faq.md"/>
         <topic id="Help_and_Support" name="Help and Support" filename="help_and_support.md"/>
     </topic>
-    <topic id="Getting_Started" name="Getting Started" filename="getting_started.md"/>
+    <topic id="Getting_Started" name="Getting Started" filename="getting_started.md">
+        <topic id="Creating_Your_Project" name="Creating your Project" filename="creating_your_project.md">
```
Comment (tomspilman) [inline Documentation/config.xml]: @dellis1972 - Missing the `/>` at the end of this topic tag.

**Comment by tomspilman** [comment]
@dellis1972 - I'll follow this PR up with my console info doc.

File: Documentation/creating_your_project.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+1. Cross Platform Desktop Project (DesktopGL) - Use this to target Windows, Mac and Linux.
+   It used OpenGL and OpenAL.
+2. Windows - Windows Only Desktop project using DirectX.
+3. Android - Targets the Android Operating System. Uses OpenGL and OpenAL
+4. iOS - Targets iPhone and iPad. Uses OpenGL and OpenAL
+4. tvOS - Targets tvOS. Uses OpenGL and OpenAL
+5. Windows Universal App (XAML) - Targets Windows 10 UWP/Xbox One. Has a XAML entry point
+   which allows you to overlay controls on top of the game screen.
+6. Windows Universal App - Targets Windows QO UWP/Xbox One. Is a pure DirectX Application
```
Comment (Jjagg) [inline Documentation/creating_your_project.md]: Windows 10

File: Documentation/creating_your_project.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+the default templates which are available Visual Studio. If you are using MacOS or Linux
+you can use MonoDevelop/Xamarin Studio to create a new Project. The template should be
+installed by default by the installer
+
+1. Open Visual Studio
+2. File->New
+3. Look for MonoGame in the list of template items available.
+4. Pick the template type you want to use.
+5. Fill in the new project Name and destination directory.
+6. Clock OK/Create.
```
Comment (Jjagg) [inline Documentation/creating_your_project.md]: Click

File: Documentation/creating_your_project.md [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+
+This will generate a default template. It will contain a `Game1.cs` file which is the
+main clas for your game. In addition there is a platform entry point. This is `Program.cs`
```
Comment (Jjagg) [inline Documentation/creating_your_project.md]: main class

File: Documentation/creating_your_project.md [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+for most platforms. For android it will be `Activity1.cs`, iOS ``.
```
Comment (Jjagg) [inline Documentation/creating_your_project.md]: Android

File: Documentation/creating_your_project.md [hunk 6]
```diff
@@ -0,0 +1,88 @@
+#If you used the Installer
+
+The installer will install templates into the IDE (if it is supported). The most common
+templates are.
+
+1. Cross Platform Desktop Project (DesktopGL) - Use this to target Windows, Mac and Linux.
```
Comment (Jjagg) [inline Documentation/creating_your_project.md]: Cross-platform

File: Documentation/creating_your_project.md [hunk 7]
```diff
@@ -0,0 +1,88 @@
...
+   It used OpenGL and OpenAL.
+2. Windows - Windows Only Desktop project using DirectX.
+3. Android - Targets the Android Operating System. Uses OpenGL and OpenAL
+4. iOS - Targets iPhone and iPad. Uses OpenGL and OpenAL
+4. tvOS - Targets tvOS. Uses OpenGL and OpenAL
+5. Windows Universal App (XAML) - Targets Windows 10 UWP/Xbox One. Has a XAML entry point
+   which allows you to overlay controls on top of the game screen.
+6. Windows Universal App - Targets Windows QO UWP/Xbox One. Is a pure DirectX Application
+   not XAML support.
```
Comment (Jjagg) [inline Documentation/creating_your_project.md]: not => without seems better

File: Documentation/creating_your_project.md [hunk 8]
```diff
@@ -0,0 +1,88 @@
...
+
+Some templates might not be available in some IDE's. For example the UWP templates are 
+only available in Visual Studio on Windows.
+
+## Visual Studio
+
+There are a number of way to go about creating a new Game. The quickest is to use one of 
+the default templates which are available Visual Studio. If you are using MacOS or Linux
```
Comment (Jjagg) [inline Documentation/creating_your_project.md]: which are available **in** Visual Studio

File: Documentation/creating_your_project.md [hunk 9]
```diff
@@ -0,0 +1,88 @@
...
+you can use MonoDevelop/Xamarin Studio to create a new Project. The template should be
+installed by default by the installer
```
Comment (Jjagg) [inline Documentation/creating_your_project.md]: Forgot the '.' at the end.

File: Documentation/creating_your_project.md [hunk 10]
```diff
@@ -0,0 +1,88 @@
...
+3. Look for MonoGame in the list of template items available.
+4. Pick the template type you want to use.
+5. Fill in the new project Name and destination directory.
+6. Clock OK/Create.
+
+This will generate a default template. It will contain a `Game1.cs` file which is the
+main clas for your game. In addition there is a platform entry point. This is `Program.cs`
+for most platforms. For android it will be `Activity1.cs`, iOS ``. 
+
+Youre new project will also contain a `Content.mgcb` file. This is the file you use to
```
Comment (Jjagg) [inline Documentation/creating_your_project.md]: Your

File: Documentation/creating_your_project.md [hunk 11]
```diff
@@ -0,0 +1,88 @@
...
+
+
+## Visual Studio for Mac
+
+For Visual Studio for Mac, a new template system is being developed. This is available 
+in the `Extensions` menu item. You will need to enable the Repository Alpha channel in
+order to install the `MonoGame.Templates` Extension.
+
+
+#If you  want to use the Nuget Packages.
```
Comment (Jjagg) [inline Documentation/creating_your_project.md]: Casing should be **NuGet**

File: Documentation/creating_your_project.md [hunk 12]
```diff
@@ -0,0 +1,88 @@
...
+5. Add the Package. 
+
+## Hooking up the Content Build System when using Nuget.
+
+The content build system is not distributed as a Nuget. It will be installed as part 
+of the Installer. However there are some additional `msbuild` imports which are required
+if you want your content to automatically build and be included in your project. 
+
+By not adding these imports you will need to manaully build your content for the platform
+you are targetting and link the .xnb files manually. Its much easier to hook up the
```
Comment (Jjagg) [inline Documentation/creating_your_project.md]: targeting. It's

File: Documentation/creating_your_project.md [hunk 13]
```diff
@@ -0,0 +1,88 @@
...
+you are targetting and link the .xnb files manually. Its much easier to hook up the 
+build system.
+
+1. Open your projects .csproj
+2. Add the following as the first element after the <Project> node.
+
+	<Import Project="$(MSBuildExtensionsPath)\MonoGame\v3.0\MonoGame.Common.props"
+		Condition="Exists('$(MSBuildExtensionsPath)\MonoGame\v3.0\MonoGame.Common.props')" />
+
+3. Add the following as the last elelemt before the </Project> node.
```
Comment (Jjagg) [inline Documentation/creating_your_project.md]: element

File: Documentation/getting_started.md [hunk 14]
```diff
@@ -1,3 +1,6 @@
 This section walks you through the basics of MonoGame, creating a simple game, and how cross-platform development works.
 
-**Add SubSection Links**
+Before you work through this section you need to setup your machine for Game development. You can follow the instructions [here](setting_up_monogame.md).
+After running through that link you will be ready to start writting your game.
```
Comment (Jjagg) [inline Documentation/getting_started.md]: writing

File: Documentation/welcome.md [hunk 15]
```diff
@@ -1,7 +1,14 @@
 Welcome to the MonoGame game library documentation hub.
 
-### WORK IN PROGRESS
-Note that the MonoGame documentation project is currently a work in progress and applies to the **latest development build** of MonoGame.
+This area of the site contains the documentation on the API of MonoGame as well as how to use it to create great games.
+Note that this is a work in progress so there will be gaps in the documentation coverage. 
+
+If you cannot find what you need here you can also look at the [Microsoft XNA documentation](). MonoGame is API compatible
```
Comment (Jjagg) [inline Documentation/welcome.md]: Link to XNA docs: https://msdn.microsoft.com/en-us/library/bb200104.aspx
And API reference: https://msdn.microsoft.com/en-us/library/bb203940.aspx
Not sure which is more appropriate.

File: Documentation/welcome.md [hunk 16]
```diff
@@ -1,7 +1,14 @@
...
+with XNA even down to the namespaces. So usually what works for XNA will work for MonoGame too. 
+
+Note that this documentation hub is built from the source code on every commit to the development branch. As such 
+it applies to the development builds available on the [Downloads](http://www.monogame.net/downloads) page. This may include new features which may
+not be available in the current stable release
```
Comment (Jjagg) [inline Documentation/welcome.md]: Forgot the '.' at the end.

File: Documentation/what_is_monogame.md [hunk 17]
```diff
@@ -1,18 +1,25 @@
...
-   * PlayStation 4
-   * PlayStation Vita
-   * Xbox One (both UWP and XDK)
-   * Nintendo Switch
- * Other
+   * Windows Phone 10
+* Television
    * tvOS
+
+MonoGame also supports a number of Game Consoles. The tempalates and source for these platforms
```
Comment (Jjagg) [inline Documentation/what_is_monogame.md]: templates

File: Documentation/what_is_monogame.md [hunk 18]
```diff
@@ -1,18 +1,25 @@
...
+are not publicly availalbe. However they are available to developers registered with the appropriate
```
Comment (Jjagg) [inline Documentation/what_is_monogame.md]: available

**Comment by Jjagg** [comment]
Oh wow, I just now saw the '[WIP Ignore spelling for now :) ]' part. I hope you don't mind

**Comment by cra0zy** [comment]
#4559 :(

File: Documentation/what_is_monogame.md [hunk 19]
```diff
@@ -1,18 +1,25 @@
...
    * tvOS
+
+MonoGame also supports a number of Game Consoles. The templates and source for these platforms
+are not publicly availalbe. However they are available to developers registered with the appropriate
+developer programs.  
+
+ * Consoles (for registered developers)
+   * PlayStation 4 (Sony)
+   * PlayStation Vita (Sony)
+   * Xbox One (both UWP and XDK) (id@xbox)
```
Comment (Jjagg) [inline Documentation/what_is_monogame.md]: Maybe put Xbox One UWP in the above section, since it's open to anyone

**Pull request merged**